ddpui/api/user_org_api.py

```diff
@@ -572,6 +574,11 @@ def post_organization_v1(request, payload: OrgSchema):
     # create a new orguser if the org is already there
     orguserfunctions.ensure_orguser_for_org(orguser, org)
 
+    # create a new orgplan
+    org_plan, error = orgfunctions.create_orgPlan(payload, org)
```

Comment:
don't mix snake_case with camelCase.. call it `create_org_plan`

Reply:
Done